src/map/utils/petutils.cpp

```diff
@@ -1096,8 +1096,10 @@ namespace petutils
 
         Pet_t* PPetData = g_PPetList.at(PetID);
 
-        if (PMaster->objtype == TYPE_PC)
+        if (PMaster->objtype == TYPE_PC){
```

Comment:
Yep, Should be all good. My wyvern hasn't disappeared on me while zoning or logging out yet. ^^b